docs/about-mono/languages/index.md

```diff
@@ -14,14 +14,14 @@ Mono-compatible compilers
 C#
 ---
 
-The main C# compiler of the Mono Project is [**mcs**](/docs/about-mono/languages/csharp/). It covers all the features in C# 3.0 (2.6) and 4.0 (preview), including 3.0 Linq and 4.0 dynamic. As of today, 'gmcs' is the default compiler based on 2.0 runtime profile, and 'dmcs' is for 4.0 runtime profile.
+The main C# compiler of the Mono Project is [**mcs**](/docs/about-mono/languages/csharp/). It covers all the features in C# 3.0 (2.6), 4.0, 5.0 and 6.0 (preview), including 3.0 LINQ, 4.0 dynamic and 5.0 async-await support. As of today, 'gmcs' is the default compiler based on 2.0 runtime profile, and 'dmcs' is for 4.0 runtime profile.
```

Comment:
6.0 isn't in preview anymore. And gmcs/dmcs were unified in mcs, so I'd just remove that sentence.